src/protocol/mod.rs

```diff
@@ -50,6 +50,11 @@ pub struct WebSocketConfig {
     /// be reasonably big for all normal use-cases but small enough to prevent memory eating
     /// by a malicious user.
     pub max_frame_size: Option<usize>,
+    /// If set to true it will allow the websocket server to accept unmasked frames from client.
+    /// Even though this behaviour is not in compliance with RFC 6455 (which requires the server
+    /// to close the connection when unmasked frame from client is received) it might be handy in some cases
+    /// as there are existing applications sending unmasked client frames.
+    pub server_allow_unmasked: Option<bool>,
```

Comment:
I seems to me that this doesn't have to be an `Option` - the default should be clear from the `Default` implementation. While all other fields are options, they seem to be so to represent "unlimited".

Reply:
I wasn't sure about this as well and rather wrapped it into Option to keep it consistent. Once you confirm I will make it into simple bool.